src/Protector.php

```diff
@@ -158,54 +163,48 @@ public function importDump(string $sourceFilePath, array $options): bool
                     echo $output->fetch();
                 }
             }
-
-            return true;
         } catch (Exception) {
-            return false;
+            $success = false;
         }
+
+        unlink($tempFilePath);
+
+        return $success;
     }
 
     /**
-     * Public function to create the Destination File Path for the dump.
+     * Public function to create the relative file path for the dump.
      *
      * @param string $fileName
-     * @param string|null $subFolder
      * @return string
      */
-    public function createDestinationFilePath(string $fileName, ?string $subFolder = null): string
+    public function getDumpFilePath(string $fileName): string
     {
         return implode(DIRECTORY_SEPARATOR, array_filter([
```

Comment:
why array filter if you're not filtering?

Reply:
We are still filtering in case there are empty occurrences.

Reply:
ah okay cool, didnt know it does that